lib/msal-browser/src/index.ts

```diff
@@ -94,6 +94,9 @@ export {
     SignedHttpRequestOptions,
 } from "./crypto/SignedHttpRequest";
 
+export { base64Encode as Base64Encode } from "./encode/Base64Encode";
```

Comment:
We can probably avoid this if we generate or stringify the JSON later. This will however help with any encoding needs that may arise later in 1P. Let me know what you folks think about it and I can do changes accordingly.
cc @konstantin-msft  and @tnorling

Reply:
I do not mind either way but we cannot remove these later as it will be public surface.

Reply:
Would prefer not to add these types of things to the public surface if we can avoid it. Assume you want to use this in 1P? For what purpose?

Reply:
`reqCnfData` generation. We can also pass the `kid` as is and encode it while constructing the server request. I need to update the 1P PR and this one to reflect it. Do we not foresee any encoding/decoding needs in 1P? Then we can go that route.

Reply:
I'd prefer that approach. We can discuss future needs when they come up, I don't forsee any right now